core/modules/system/system.module

```diff
@@ -86,6 +86,9 @@ function system_theme() {
     'status_report' => array(
       'render element' => 'requirements',
     ) + $base,
+    'admin_info' => array(
+      'variables' => array('label' => NULL, 'machine' => NULL, 'inline' => TRUE),
+    ) + $base,
```

Comment:
The name "admin_info" seems ambiguous. How about "machine_name_label"?

Also, I think everywhere this is called it should be done with a theme suggestion (double-underscore), e.g. `theme('machine_name_label__node_type')` or `theme('machine_name_label__menu')`

Reply:
good ideas. done.